core/backend/src/IModelDb.ts

```diff
@@ -425,7 +425,7 @@ export abstract class IModelDb extends IModel {
   /** Return `true` if the underlying nativeDb is open and valid.
    * @internal
    */
-  public get isOpen(): boolean { return undefined !== this.nativeDb; }
+  public get isOpen(): boolean { return undefined !== this.nativeDb && this.nativeDb.isOpen(); }
 
```

Comment:
You can remove the check for undefined

Reply:
This is because of EC Presentation mock test. where nativeDb is undefined. so isOpen throw an exception. 

I will update their test and remove this.

Reply:
doesn't look fixed to me?